cmd/juju/controller/unregister.go

```diff
@@ -92,8 +92,8 @@ func (c *unregisterCommand) Init(args []string) error {
 
 var unregisterMsg = `
 This command will remove connection information for controller %q.
-Doing so will prevent you from accessing this controller until
-you register it again.
+This action will not destroy the controller but will prevent you from accessing it
```

Comment:
"The controller will not be destroyed but you must register it again if you want to access it."

The wording change above avoids "This command ... This action..." stutter.

Reply:
That does sounds better, thanks Ian :D